File: src/main/java/eu/virtusdevelops/easyclans/dao/ClanDao.java
package eu.virtusdevelops.easyclans.dao;

import eu.virtusdevelops.easyclans.models.Clan;
import eu.virtusdevelops.easyclans.models.Log;

import java.util.List;
import java.util.UUID;

public interface ClanDao extends DaoCrud<Clan, UUID> {

    List<Log> getClanLogs(UUID id, int amount, int page);

    List<Log> getPlayerClanLogs(UUID id, UUID playerId, int amount, int page);
}

File: core/src/main/kotlin/eu/virtusdevelops/easyclans/core/clan/ClanDao.kt
package eu.virtusdevelops.easyclans.core.clan

import eu.virtusdevelops.easyclans.api.clan.Clan
import eu.virtusdevelops.easyclans.core.database.DaoCrud
import java.util.UUID

interface ClanDao : DaoCrud<Clan, UUID> {



}